src/openrct2/rct12/RCT12.cpp

```diff
@@ -752,6 +752,13 @@ void RCT12PathElement::SetIsBlockedByVehicle(bool isBlocked)
     }
 }
 
+bool RCT12TrackTypeIsBooster(uint8_t rideType, uint16_t trackType)
```

Comment:
`RCT2TrackTypeIsBooster`. Not "RCT12...". RCT1 always has a booster at index 100, so does not need conversion.